### DISCUSSION TITLE
NRF installation without docker query

### DISCUSSION THREAD

**Discussion by Deblina96** [OAI/OpenAPI-Specification#3471] [Q&A]
I am only installing NRF using https://gitlab.eurecom.fr/oai/cn5g/oai-cn5g-nrf/-/wikis/Installation this without docker.
My question is that does this installation also install mysql DB also with it ? if yes then where does it get installed ?

**Comment by handrews** [comment]
@Deblina96 this repository is for the OpenAPI specification.  You'll need to contact whoever is providing that software to get an answer to your question.
